05/app.js

```diff
@@ -0,0 +1,50 @@
+//ZROBIONE
+//UWAGI: wydaje mi się, e jednak potrzebna będzie walidacja wszystkich inputów (required)
+// i nie będzie potrzebne sprawdzanie imienia i nazwiska
+// czyli zostawiamy tylko sprawdzenie kodu pocztowego - czy zgodny z patternem
```

Comment:
Należy dodać do formularza atrybut "novalidate", który powoduje, że przeglądarka nie wykonuje sprawdzenia (waliadcji) i my musimy to zrobić. Dlatego powinniśmy sprawdzać wszystkie elementy. Opcja sprawdzania przez przeglądarkę jest nie wystarczająca, bo każdy może ją wyłączyć więc nie możemy ufać temu mechanizmowi i wszystko powinniśmy sprawdzić "osobiście".

Reply:
ok